lib/rubocop/cop/betterment/boolean_query_parameter.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      class BooleanQueryParameter < Base
+        MSG = <<~MSG
+          A boolean fetched from query params will never be false when
```

Comment:
I think we should make this linter look for `ENV` in addition to `params` (cc @samandmoore since you hit this with ENV recently)

Reply:
I should add some more variance in the test cases to clarify this, but to be clear we are not looking at the name of the var/method that fetch is called on, but actually just anything that fetch gets called on inside a controller. So `ENV.fetch(:SHOULD_RUN, false)` would be covered inside a controller. Maybe we want a special exception to handle ENV outside of controllers too?

Reply:
yea we should handle ENV, in this regard, everywhere. because i ran into it in some ruby_spec_helpers setup / config code

Reply:
Yeah, to be clear, we'd want `params.fetch()` within controllers, and `ENV.fetch()` anywhere.

Reply:
Got a chance to revisit this. I have made it check for ENV.fetch in all places